packages/react-data-grid/src/Row.js

```diff
@@ -157,7 +157,7 @@ const Row = React.createClass({
   getExpandableOptions(columnKey) {
     let subRowDetails = this.props.subRowDetails;
     if (subRowDetails) {
-      return { canExpand: subRowDetails && subRowDetails.field === columnKey && subRowDetails.children && subRowDetails.children.length > 0, field: subRowDetails.field, expanded: subRowDetails && subRowDetails.expanded, children: subRowDetails && subRowDetails.children, treeDepth: subRowDetails ? subRowDetails.treeDepth : 0, subRowDetails: subRowDetails };
+      return { canExpand: subRowDetails && subRowDetails.field === columnKey && ((subRowDetails.children && subRowDetails.children.length > 0) || subRowDetails.group === true), field: subRowDetails.field, expanded: subRowDetails && subRowDetails.expanded, children: subRowDetails && subRowDetails.children, treeDepth: subRowDetails ? subRowDetails.treeDepth : 0, subRowDetails: subRowDetails };
```

Comment:
this function determine the expandableOptions property that get passed to Cell as props.

Need test to verify these expandable options based on subRowDetails prop

eg. canExpand
field
children
treeDepth